extensions/tiled-assets/examples/example-dimension.json

```diff
@@ -0,0 +1,220 @@
+{
+    "stac_version": "0.9.0",
+    "stac_extensions": [
+        "datacube",
+        "eo",
+        "tiles"
```

Comment:
The value to use here (and probably also in all other examples) is the identifier/folder name.

Reply:
👍